code/modules/mob/living/carbon/inventory.dm

```diff
@@ -44,7 +44,8 @@
 			var/mob/dead/observe = M
 			if(observe.client)
 				observe.client.screen -= I
-	I.forceMove(src)
+	if(I.loc != src)
+		I.forceMove(src)
```

Comment:
What's the point of this, shouldn't `forceMove` be the one that cares if locs are equal, not the consumer?

Reply:
seems like it does

Reply:
gone now